packit/api.py

```diff
@@ -2777,3 +2777,76 @@ def get_vm_image_build_status(
             refresh_token=self.config.redhat_api_refresh_token,
         )
         return ib.get_image_status(build_id)
+
+    def run_local_test(
+        self,
+        target: Optional[str] = "fedora:rawhide",
+        rpm_paths: Optional[list[str]] = None,
+        run_all: bool = False,
+        plans: Optional[list[str]] = None,
+    ) -> Optional[str]:
+        """
+        Run tests locally via tmt.
+        """
+
+        if not rpm_paths:
+            raise PackitException("At least one --rpm_path is required")
+
+        if run_all and plans:
+            raise PackitException("Only one of --run-all, or --plans may be used.")
```

Comment:
Not correct, I don't have to pass anything, and `run_all` defaults to `False`

Reply:
Now that I think about it, I've never used `--all` when running `tmt`, what's it for?